git_repo/git_repo.go

```diff
@@ -38,6 +38,21 @@ func RepoLocatedAt(path string) GitRepo {
 	return GitRepo{absoluteRoot}
 }
 
+//Gets all the staged files and collects the diff section in each file
+func (repo GitRepo) GetDiffForStagedFiles() []Addition {
+	files := repo.stagedFiles()
+	result := make([]Addition, len(files))
+	for i, file := range files {
+		data := repo.fetchStagedDiff(file)
+		result[i] = NewAddition(file, data)
+	}
+
+	log.WithFields(log.Fields{
+		"additions": result,
```

Comment:
Let's try and make logging of result less verbose.
We can
* Make this a debug log instead of info
or 
* Make sure the member Data []byte of type Addition is not logged by default